packages/frontend/src/providers/ThirdPartyServicesProvider.tsx

```diff
@@ -1,10 +1,50 @@
 import { PostHogProvider, usePostHog } from 'posthog-js/react';
-import { type FC } from 'react';
+import { useEffect, type FC } from 'react';
+import { Helmet } from 'react-helmet';
 import { IntercomProvider } from 'react-use-intercom';
 import { Intercom } from '../components/Intercom';
 import useSentry from '../hooks/thirdPartyServices/useSentry';
 import { useApp } from './AppProvider';
 
+const Pylon = () => {
+    const { user, health } = useApp();
+    useEffect(() => {
+        if (health.data?.pylon?.appId && user.data) {
```

Comment:
what do you think of adding this `health.data.pylon.verificationHash` to the condition?

```suggestion
        if (health.data?.pylon?.appId && health.data.pylon.verificationHash && user.data) {
```

Reply:
Pylon's identity verification is optional. So it can work without the `verificationHash`